picard/ui/metadatabox.py

```diff
@@ -317,13 +329,21 @@ def contextMenuEvent(self, event):
         menu.exec_(event.globalPos())
         event.accept()
 
+    def add_to_preserved_tags(self, name, preserved_tags):
+        preserved_tags.append(name)
+        config.setting['preserved_tags'] = ", ".join(uniqify(preserved_tags))
+
+    def remove_from_preserved_tags(self, name, preserved_tags):
+        preserved_tags = filter(lambda x: x != name, preserved_tags)
+        config.setting['preserved_tags'] = ", ".join(uniqify(preserved_tags))
```

Comment:
This small code duplication (line 334 & 338) could be solved using a small static method.